serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,83 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        urls = [self.URL.format(year) for year in range(2008, 2018)]
```

Comment:
Ok, I made a constant for first and next year :)
Adapted to what you suggested, soon will be pushing it.